server/src/test/java/org/elasticsearch/common/regex/RegexTests.java

```diff
@@ -63,4 +63,24 @@ public void testDoubleWildcardMatch() {
         assertTrue(Regex.simpleMatch("fff*******ddd", "fffabcddd"));
         assertFalse(Regex.simpleMatch("fff******ddd", "fffabcdd"));
     }
+
+    public void testSimpleMatch() {
+        for (int i = 0; i < 100000; i++) {
```

Comment:
Leftover from benchmarking experiments?

Reply:
Meh 100k iterations is still only 300ms :)

Reply:
Reduced in 489e82b.